frontend/components/Header/Navbar.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from "react";
+import Link from "next/link";
+import { MenuIcon } from "@heroicons/react/outline";
+
+export default function Navbar() {
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  const handleClick = () => {
+    setMenuOpen(!menuOpen);
+  };
+
+  return (
+    <div className="relative flex w-full flex-col items-end justify-end sm:flex-row">
+      <div className="mx-4 flex h-8 w-8 items-center sm:hidden">
+        <button id="header__toggler" onClick={handleClick} className="w-full">
+          <MenuIcon></MenuIcon>
+        </button>
+      </div>
+
+      <div
+        id="header__menu-items"
+        className={`${
+          menuOpen ? "flex flex-col" : "hidden"
+        } absolute top-9 items-end space-x-4 bg-white p-4 text-lg font-bold sm:static sm:mr-4 sm:flex sm:flex-row sm:p-0`}
+      >
+        <Link href="/wiki" target="_blank" rel="noreferrer noopener">
+          <a className="w-full p-2 text-right hover:border-b-2 sm:p-0">Wiki</a>
```

Comment:
Adding a border on hover causes the menu texts to jump vertically:

https://user-images.githubusercontent.com/4383/189989723-89362268-23ec-4028-9af1-1e395c5dfc70.mp4

Two options come to mind:

1. Have a transparent border be visible always, and change its colour to grey when hovering:

   ```
   border-b-2 border-b-transparent hover:border-b-gray-300
   ```

2. Use an underline and set the underline width, offset, and colour.

   ```
   hover:underline hover:decoration-gray-300 hover:decoration-2 hover:underline-offset-4
   ```

I'd lean towards the second option since the first causes the text to be offset vertically. It's slightly closer to the top of the menu than the bottom when using a border.

Here's what it looks like with an underline:

https://user-images.githubusercontent.com/4383/189989530-c0ff648a-97ee-483f-91ba-f3b622cf7a47.mp4

Reply:
Good that you spotted this, thanks! I agree with you on the second solution, I've added this to the branch.